src/commands/build.ts

```diff
@@ -45,11 +45,12 @@ export class BuildCommand implements Command {
           'used for service worker generation.'
     },
     {
-      name: 'insert-dependency-links',
+      name: 'add-prefetch',
```

Comment:
I'm not really concerned about long flag names for things that'll hopefully be in a config file, if they're more descriptive. I think `insert-prefetch-links` is the most clear name.